-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.063 s <<< FAILURE! - in TestSuite
three(com.test.cases.Demo)  Time elapsed: 0.005 s  <<< FAILURE!
java.lang.AssertionError: expected [2] but found [1]
	at com.test.cases.Demo.three(Demo.java:34)

